config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get '/items', to: 'items#index'
+
+  get '/customers/:id', to: 'customers#show'
+  post '/customers/:id', to: 'customers_items#create'
```

Comment:
I left this comment in your view, but 1) your controller organization is great and very RESTful in that you put the `create` action in a joins controller. But 2) the path you used to route to that action could be more RESTful. Consider: "/customers/:id/items"

Reply:
This doesn't impact your score at all, is more just an enhancement